### ISSUE TITLE
incorrect uses of Bound2.dilated

### ISSUE THREAD

**Issue by pixelzoom** [phetsims/ph-scale#181]
For phetsims/QA#514 and phetsims/QA#515. 

Bounds2.dilated take 1 argument:

```js
  dilated: function( d ) {
    return new Bounds2( this.minX - d, this.minY - d, this.maxX + d, this.maxY + d );
  },
```

There are a few cases where it's being called with 2 arguments.

```js
// ZoomButtonGroup.js:
    zoomOutButton.touchArea = zoomOutButton.localBounds.dilated( 5, 5 );
    zoomInButton.touchArea = zoomOutButton.localBounds.dilated( 5, 5 );

// In MacroPHMeterNode.js:
   this.touchArea = this.localBounds.dilated( 20, 20 );
```

All of the above should have 1 argument.   This doesn't hurt anything, because the 2nd arg is ignored.

**Comment by pixelzoom** [comment]
Fixed in master, cherry-picked to 1.4 branches of ph-scale and ph-scale-basics.   

Labeling with "fix-awaiting-deploy".  There's nothing to test here, but I want to remember that the change has been made.

**Comment by pixelzoom** [comment]
This has shown no problems in many cycles of CT, and there is nothing specific to test.  So closing.
